components/x-audio/src/redux/swipe-down-to-minimise.js

```diff
@@ -0,0 +1,31 @@
+const EXPANDED_PLAYER_HEIGHT = 360;
+const DISPLAY_EXPANDED_THRESHOLD = EXPANDED_PLAYER_HEIGHT / 2;
+
+function setPositionY (target, posY) {
+	target.style.transform = `translate3d(0, ${posY}px, 0)`;
+}
+
+export default (event, playerActions) => {
```

Comment:
so this file doesn't need to be aware of redux, could pass in an `onEnd` function?

Reply:
👍